config/rootfs/debos/overlays/fluster/usr/bin/fluster_parser.py

```diff
@@ -0,0 +1,106 @@
+#!/usr/bin/env python3
```

Comment:
Please add a copyright header with license information like the other Python files.

Reply:
Done, thanks for the heads up!